packages/systemd-257/systemd-257.spec

```diff
@@ -0,0 +1,799 @@
+Requires: %{name}
+Provides: %{_cross_os}systemd-console = %{package_priority_epoch}:
+
+%description console
+%{summary}.
+
+%package cryptsetup
+Summary: Files for cryptsetup support in systemd
+Requires: %{name}
+Requires: %{_cross_os}cryptsetup
+Provides: %{_cross_os}systemd-cryptsetup = %{package_priority_epoch}:
+
+%description cryptsetup
+%{summary}.
+
+%package devel
+Summary: Files for development using the System and Service Manager
+Requires: %{name}
+Provides: %{_cross_os}systemd-devel = %{package_priority_epoch}:
+
+%description devel
+%{summary}.
+
+%package networkd
+Summary: Files for networkd
+Requires: %{name}
+Provides: %{_cross_os}systemd-networkd = %{package_priority_epoch}:
+
+%description networkd
+%{summary}.
+
+%package resolved
+Summary: Files for resolved
+Requires: %{name}
+Provides: %{_cross_os}systemd-resolved = %{package_priority_epoch}:
+
+%description resolved
+%{summary}.
+
+%prep
+%autosetup -n systemd-%{version} -p1
+
+%build
+CONFIGURE_OPTS=(
+ -Dmode=release
+
+ -Dsplit-bin=true
+ -Dlink-udev-shared=true
+ -Dlink-systemctl-shared=true
+ -Dlink-networkd-shared=false
+ -Dlink-timesyncd-shared=false
+ -Dlink-boot-shared=false
+ -Dstatic-libsystemd=false
+ -Dstatic-libudev=false
+
+ -Dsysvinit-path=''
+ -Dsysvrcnd-path=''
+ -Dinitrd=false
+ -Dnscd=false
+
+ -Dutmp=false
+ -Dhibernate=false
+ -Dldconfig=true
+ -Dresolve=true
+ -Defi=true
+ -Dtpm=false
+ -Denvironment-d=false
+ -Dbinfmt=true
+ -Drepart=enabled
+ -Dcoredump=false
+ -Dpstore=true
+ -Doomd=false
+ -Dlogind=true
+ -Dhostnamed=false
+ -Dlocaled=false
+ -Dmachined=false
+ -Dportabled=false
+ -Dsysext=false
+ -Dsysupdate=disabled
+ -Duserdb=false
+ -Dhomed=disabled
+ -Dnetworkd=true
+ -Dtimedated=false
+ -Dtimesyncd=false
+ -Dremote=disabled
+ -Dnss-myhostname=false
+ -Dnss-mymachines=disabled
+ -Dnss-resolve=enabled
+ -Dnss-systemd=false
+ -Dfirstboot=false
+ -Drandomseed=true
+ -Dbacklight=false
+ -Dvconsole=false
+ -Dquotacheck=false
+ -Dsysusers=true
+ -Dtmpfiles=true
+ -Dimportd=disabled
+ -Dhwdb=false
+ -Drfkill=false
+ -Dxdg-autostart=false
+ -Dman=disabled
+ -Dhtml=disabled
+ -Dtranslations=false
+ -Dgpt-auto-generator=false
+ -Dlog-message-verification=disabled
+
+ -Dcertificate-root='%{_cross_sysconfdir}/ssl'
+ -Dpkgconfigdatadir='%{_cross_pkgconfigdir}'
+ -Dpkgconfiglibdir='%{_cross_pkgconfigdir}'
+
+ -Dadm-group=false
+ -Dwheel-group=false
+
+ -Dgshadow=true
+
+ -Ddefault-dnssec=no
+ -Ddefault-dns-over-tls=no
+ -Ddefault-mdns=no
+ -Ddefault-llmnr=no
+ -Ddns-over-tls=false
+ -Ddns-servers=""
+
+ -Dsupport-url="https://github.com/bottlerocket-os/bottlerocket/discussions"
+
+ -Dseccomp=auto
+ -Dselinux=auto
+ -Dapparmor=disabled
+ -Dsmack=false
+ -Dpolkit=disabled
+ -Dima=false
+
+ -Dacl=enabled
+ -Daudit=disabled
+ -Dblkid=enabled
+ -Dfdisk=enabled
+ -Dkmod=enabled
+ -Dpam=disabled
+ -Dpasswdqc=disabled
+ -Dpwquality=disabled
+ -Dmicrohttpd=disabled
+ -Dlibcryptsetup=disabled
+ -Dlibcryptsetup-plugins=disabled
```

Comment:
Can you move all the cryptsetup and TPM options to their own block, so it's clear what will need to be flipped to turn it back on?